.github/workflows/ci.yml

```diff
@@ -59,19 +76,11 @@ jobs:
       run: |
         go build -trimpath -a -ldflags="-w -s" -v
 
-    - name: Publish Build Artifact (Windows)
-      if: matrix.os == 'windows-latest'
+    - name: Publish Build Artifact
       uses: actions/upload-artifact@v1
       with:
         name: caddy_v2_${{ matrix.os }}
-        path: ./cmd/caddy/caddy.exe
-
-    - name: Publish Build Artifact (Linux/Mac)
-      if: matrix.os != 'windows-latest'
-      uses: actions/upload-artifact@v1
-      with:
-        name: caddy_v2_${{ matrix.os }}
-        path: ./cmd/caddy/caddy
+        path: ${{ matrix.CADDY_BIN_PATH }}
```

Comment:
It is _kind_ of a bummer that the artifacts names are all the same ("latest") so if one is downloaded, one can quickly forget which commit it's built from (since `caddy version` shows `(devel)`).

1. Would it be difficult to add the first 7-8 chars of the commit SHA to the filename, instead of `latest`? (Also, `ubuntu` -> `linux` maybe?)
2. And/or would it be difficult to make `caddy version` show the proper version?